sdk/src/beta9/runner/function.py

```diff
@@ -227,6 +227,7 @@ def invoke_function(
             callback_url=callback_url,
         )
     except BaseException as e:
+        print(f"{traceback.format_exc()}")
```

Comment:
This already happens here https://github.com/beam-cloud/beta9/pull/671/files#diff-1739866d6bf74331c67757941a077b334836efd70a493b62ba971a863e5604cbL177

Reply:
or it doesn't seem to. it's not working on my end.